src/.gitignore

```diff
@@ -1,3 +1,7 @@
 .idea/
 .vscode/
-node_modules/
\ No newline at end of file
+node_modules/
+templates/*/*/chapters/
```

Comment:
I looked into it and you can’t have different `.gitignores` for each branch [without hacks workarounds](https://stackoverflow.com/questions/29579546/git-excludesfile-for-a-branch/29583813#29583813).

We could update `.gitignore` as part of this but that feels hacky too.

Ultimately we need to decide if storing in git or not. I don’t see a point in half storing it.

Btw I’m still not 100% convinced about not storing it. It’s nice to be able to see HTML differences for some changes. Then again the actual HTML rather than Jinja2 template is what’s important and we don’t store that - though maybe should if we decide to go static site.

For now on balance I’m still for removing it. Not 100% but maybe 90% 😀

Reply:
The script now zips to deploy.zip which can be uploaded to Release folder as a release artifact. Might look to automate this more in the future.